### PR TITLE
feat(slugrunner): make slugrunner configurable via env variables

### PR DESCRIPTION
**Pull request by smothiki**
Needs to change deis-dev charts workflow-rc yaml to support immutable slugrunner image

### CODE REVIEW

**Comment by helgi** [comment]
Fixes #135

File: rootfs/deis/settings.py [hunk 1]
```diff
@@ -262,6 +262,8 @@
 random_secret = 'CHANGEME_sapm$s%upvsw5l_zuy_&29rkywd^78ff(qi*#@&*^'
 SECRET_KEY = os.environ.get('DEIS_SECRET_KEY', random_secret)
 BUILDER_KEY = os.environ.get('DEIS_BUILDER_KEY', random_secret)
+defaultImage = "quay.io/deisci/slugrunner:v2-beta"
```
Comment (kmala) [inline rootfs/deis/settings.py]: shouldn't the default be quay.io/deisci/slugrunner:canary

Comment (smothiki) [inline rootfs/deis/settings.py]: I'll change it asap

**Comment by smothiki** [comment]
Closing this PR and will open an updated one . As not able to update commits to this PR because of renaming workflow to controller

**Pull request closed without merging**